helm-chart/values.yaml

```diff
@@ -319,62 +322,58 @@ grafana:
       enabled: false
 
 ingress:
-  enabled: true
-  annotations:
-    traefik.ingress.kubernetes.io/router.entrypoints: web
-    kubernetes.io/ingress.class: traefik
-  hosts:
-    - host: cvat.local
-      paths:
-        - path: /api
-          pathType: "Prefix"
-          service:
-            name: backend-service
-            port: 8080
-        - path: /admin
-          pathType: "Prefix"
-          service:
-            name: backend-service
-            port: 8080
-        - path: /static
-          pathType: "Prefix"
-          service:
-            name: backend-service
-            port: 8080
-        - path: /django-rq
-          pathType: "Prefix"
-          service:
-            name: backend-service
-            port: 8080
-        - path: /git
-          pathType: "Prefix"
-          service:
-            name: backend-service
-            port: 8080
-        - path: /opencv
-          pathType: "Prefix"
-          service:
-            name: backend-service
-            port: 8080
-        - path: /profiler
-          pathType: "Prefix"
-          service:
-            name: backend-service
-            port: 8080
-        - path : /
-          pathType: "Prefix"
-          service:
-            name: frontend-service
-            port: 80
+  enabled: false
+#  enabled: true
+#  annotations:
```

Comment:
`ingress.enabled = true` does nothing, why rules should be commented by you opinion?

Reply:
Because there are some lists in that config and they are hard to override with user values files.
For example "paths" list  here will be very hard to override for me to use another Ingress provider like nginx, or ambassador, or gke lb

Reply:
my config just for example: 
<img width="299" alt="image" src="https://github.com/opencv/cvat/assets/3233532/d2493f8b-33a6-4b3e-8806-c74baf4ee8df">

Reply:
also ingress.enblaed is used here, for example: 
<img width="914" alt="изображение" src="https://github.com/opencv/cvat/assets/3233532/1bf5e20b-547d-4a45-8e9c-44d1f1bc2814">

Reply:
also ingress.enabled is used here: 
<img width="883" alt="изображение" src="https://github.com/opencv/cvat/assets/3233532/ea1a652a-8fdc-4a79-906a-6f1080dffa23">

Reply:
@azhavoro @Keramblock Maybe we should add ImplementationSpecific to default values?

Reply:
@Keramblock Could you please add comment to commented values in ingress section, smth like "In case you need an ingress, don't forget to uncomment paths, or write them manually in your values"

Reply:
@azhavoro also I moved traefik configuration to tests.values.yml just in case you will need it for your internal stuff, so that change will not interrupt you workflow.

Reply:
@azhavoro The problem with default paths with predefined host "cvat.local" that if you specify your own host and other configs for paths, then you get:
```yaml
hosts:
  - host: cvat.local
     paths:
       - path: /api
   - host: my-cvat-host.example.com
      paths:
        - path: /api
```
It is a bit ugly and contains unused values, I think it'd be better to preserve default paths maybe and allow to specify host (as basic config)

Reply:
Also, according to this docs: https://docs.nginx.com/nginx-ingress-controller/configuration/ingress-resources/basic-configuration/#new-features-available-in-kubernetes-118-and-above
ImplementationSpecific is a default value, so we can just omit it here

Reply:
@eremeevfd default values is for cvat.local host. I doubt that will be usefull for other users.

Reply:
> @azhavoro @Keramblock Maybe we should add ImplementationSpecific to default values?

default values is for cvat.local host. I doubt that will be usefull for other users.